### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by BenMorganIO**

### CODE REVIEW

**Comment by nickcharlton** [comment]
Excellent! This leaves us with Portuguese to go now!

**Comment by nickcharlton** [comment]
@sardaukar Could I get you to look at the Portuguese translations here?

File: config/locales/administrate.es.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ es:
       destroy: Destruir
       edit: Editar
       show: Mostrar
+      new: Novaj
+      back: Reen
```
Comment (nickcharlton) [inline config/locales/administrate.es.yml]: @pablobm or @ismasan Could I get one of you two to look at this Spanish locale? (Even better the whole file!)

Comment (ismasan) [inline config/locales/administrate.es.yml]: I see 2 random non-spanish words in this excerpt of the file alone :)

I can commit a fix on Monday.

Comment (ismasan) [inline config/locales/administrate.es.yml]: I've cloned Ben Morgan's fork, added a fix and pushed it to https://github.com/thoughtbot/administrate/compare/translations

Things we can do

* Fork Ben's fork and submit a PR to his branch.
* You can merge this PR and cherry-pick [my last commit](https://github.com/thoughtbot/administrate/commit/680caeeb1d83fc105c59c7f797c4fcbc10dea724) on our repo.
* Ben can cherry-pick my commit from our repo?

```
git fetch thoughtbot-origin
git cherry-pick 680caeeb1d83fc105c59c7f797c4fcbc10dea724
git push
```

File: config/locales/administrate.pt.yml [hunk 2]
```diff
@@ -7,6 +7,8 @@ pt:
       destroy: Remover
       edit: Editar
       show: Visualizar
+      new: Novo
+      back: Costas
```
Comment (jayroh) [inline config/locales/administrate.pt.yml]:
```
back: Voltar atrás
```

literally "return back" or "go back"

**Comment by BenMorganIO** [comment]
:tada:

File: config/locales/administrate.pl.yml [hunk 3]
```diff
@@ -6,6 +6,8 @@ pl:
       destroy: Usuń
       edit: Edytuj
       show: Wyświetl
+      new: Nowy
+      back: Plecy
```
Comment (krzysiek1507) [inline config/locales/administrate.pl.yml]: Almost! `plecy` is [part of body](https://www.fitomento.com/images/artykuly_fotol/Zacznij_dbac_o_swoje_plecy_trening_cwiczenia_na_plecy.jpg)

**Comment by nickcharlton** [comment]
@krzysiek1507 I would love a PR to fix it! (mostly so you can get credit).

File: config/locales/administrate.pl.yml (hunk 3 above)
Comment (krzysiek1507) [inline config/locales/administrate.pl.yml]: Will try to fix it! Probably it is `Powrót`

**Pull request merged**